core/domain/improvements_domain.py

```diff
@@ -113,7 +101,7 @@ def __init__(
             resolved_on: datetime. The datetime at which this task was resolved.
                 Only used when status is resolved, otherwise replaced with None.
         """
-        if status != improvements_models.TASK_STATUS_RESOLVED:
+        if status != constants.TASK_STATUS_RESOLVED:
```

Comment:
This is already located in `constants`? Can we also use the value from `constants` in `improvements_models`?

Reply:
Yes `TASK_STATUS_RESOLVED` is already in `constants` so I used it here. For some reason, same thing is done is improvement_models but the variables are redefined.
https://github.com/oppia/oppia/blob/6a2f0f44442fe9331bf0ed49b3d7ba8e8cd79680/core/storage/improvements/gae_models.py#L42

Reply:
I think the redefinition is not needed.

Reply:
@vojtechjelinek Should I refactor the model file then?

Reply:
I have removed the re-definitions.